faker/providers/company/it_IT/__init__.py

```diff
@@ -349,3 +367,12 @@ def bs(self):
             result.append(self.random_element(word_list))
 
         return " ".join(result)
+
+
```

Comment:
There are too many lines between those two methods. Can you remove one?

Reply:
Is it now ok with my latest commit, or do I need to do something else?